web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-studio.adoc

```diff
@@ -0,0 +1,282 @@
+= Use Studio to Configure Web Service Consumer 1.6 - Mule 4
+:page-aliases: connectors::web-service/web-service-consumer-consume.adoc
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add Consume operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure Consume operation>>.
+
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:web-service-consumer-reference.adoc[Web Service Consumer Reference].
```

Comment:
Valkyrie and I talked about this and decided to revise it to name the other topics but not link to them, moving the links to See Also at the end. The template should be updated but has not been yet:
```suggestion
If you are new to configuring connectors in Studio, see Use Studio to Configure a Connector. If, after reading this topic, you need additional information about the connector fields, see Web Service Consumer Connector Reference.
```

Reply:
You can check in with your team about this if you like; I have edited Diane's recent work similarly.